scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -96,18 +97,17 @@
                                      @(:pool->fenzo-atom compute-cluster)
                                      mesos-status))
 
-(defn synthetic-pod->job-uuid
-  "If the given pod is a synthetic pod for autoscaling, returns the job uuid
-  that the pod corresponds to (stored in a pod label). Otherwise, returns nil."
-  [^V1Pod pod]
-  (some-> pod .getMetadata .getLabels (.get api/cook-synthetic-pod-job-uuid-label)))
+(defn synthetic-pod?
+  "Given a pod name, returns true if it has the synthetic pod prefix"
+  [pod-name]
+  (str/starts-with? pod-name api/cook-synthetic-pod-name-prefix))
```

Comment:
### Why?

This allows us to identify synthetic pods even when we don't a `V1Pod` object to inspect.

See [this comment](https://github.com/twosigma/Cook/pull/1296#discussion_r379062248) for more context.